docs/MANUAL-TESTING.md

```diff
@@ -0,0 +1,114 @@
+Manual Testing
+==============
+
+This document describes a high-level script of manual tests to check for. We
+should aim to replace items on this list with automated Spectron test cases.
+
+Image Selection
+---------------
+
+- [ ] Cancel image selection dialog
+- [ ] Select an unbootable image (without a partition table), and expect a
+  sensible warning
+- [ ] Attempt to select an unsupported image
+- [ ] Change image selection
+- [ ] Select a Windows image, and expect a sensible warning
+
+Drive Selection
+---------------
+
+- [ ] Open the drive selection modal
+- [ ] Remove all drives
+- [ ] Switch drive selection
+- [ ] Insert a single drive, and expect auto-selection
+- [ ] Insert more than one drive, and don't expect auto-selection
+- [ ] Insert a locked SD Card and expect a warning
+- [ ] Insert a too small drive and expect a warning
+- [ ] Put an image into a removable drive and attempt to flash the image to the
+  drive that contains it
+- [ ] Attempt to flash a compressed image into a drive that is big enough to
+  hold the compressed image, but not big enough to hold the uncompressed
```

Comment:
The behaviour here will be different for those compression formats where we _can_ get the final uncompressed size, and those compression formats where we can't get the final uncompressed size. I'm assuming this is about testing the latter, given that Etcher doesn't allow the former?

Reply:
Done